src/analog/utils/CommandBinaryBuffer.ts

```diff
@@ -576,19 +576,19 @@ const fourChannelsBitMask = {
     channel4: Math.pow(2, 3)
 };
 
-// const byteToPulseCoefficientMap = new Map([
-//     [0x80, 1],
-//     [0x81, 5],
-//     [0x82, 10],
-//     [0x83, 100],
-//     [0x84, 1000],
-//     [0x85, 10000],
-//     [0x86, 100000]
-// ]);
+const byteToPulseCoefficientMap = new Map([
```

Comment:
we can't use Map as there are problems with otto

Reply:
reworked as objects